### PR TITLE
fix(unit-test): act() warning refactor

### PR DESCRIPTION
**Pull request by alebinson**

### CODE REVIEW

**Comment by alebinson** [comment]
@asafMasa we are pretty much clean from act() warning.
In local machine(windows) it may occur once or twice, but in pipeline it's clear

File: src/common/test-helpers/button.helper.spec.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  return wrapper
+    // @ts-ignore
+    .findWhere((n) => {
+      return n.type() === Button &&
+        n.prop('children').props['id'] === id;
+    });
+};
+
+describe.skip('Button helpers dummy suite', () => {
+  it('dummy test', () => {
```
Comment (asafMasa) [inline src/common/test-helpers/button.helper.spec.ts:15]: what is the purpose of this test?

File: src/common/test-helpers/button.helper.spec.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    expect(true).toBe(true);
+  });
+});
+/* eslint-enable */
```
Comment (asafMasa) [inline src/common/test-helpers/button.helper.spec.ts:19]: end of file

File: src/common/test-helpers/general.helper.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+//           selectedPolygon={polygon}
+//           handleExport={handleExport}
+//         />
+//       </IntlProvider>
+//     </StoreProvider>
+//   );
+//   expect(true).toBe(true);
+// });
+
+describe.skip('General helpers dummy suits', () => {
```
Comment (asafMasa) [inline src/common/test-helpers/general.helper.spec.tsx:48]: what is the purpose of this test?

File: src/common/test-helpers/text-field.helper.spec.ts [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    });
+    fieldWrapper.simulate('blur');
+  });
+
+};
+
+export const updateField = (wrapper: ReactWrapper, fieldName: string, value: number | string) => {
+  const fieldWrapper = wrapper.find(TextField).find({ name: fieldName }).find('input');
+
+  // fieldWrapper.simulate('focus');
```
Comment (asafMasa) [inline src/common/test-helpers/text-field.helper.spec.ts:44]: ?

File: src/common/test-helpers/text-field.helper.spec.ts [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+      data: value
+    },
+    // simulate changing e.target.name and e.target.value
+    target: {
+      name: fieldName,
+      value
+    },
+  });
+
+  // fieldWrapper.simulate('keyDown', {
```
Comment (asafMasa) [inline src/common/test-helpers/text-field.helper.spec.ts:57]: ?

File: src/common/test-helpers/text-field.helper.spec.ts [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+  //       fieldWrapper.simulate('blur');
+  //     },
+  //   },
+  // });
+
+  fieldWrapper.simulate('blur');
+};
+
+describe.skip('TextField helpers dummy suite', () => {
+  it('dummy test', () => {
```
Comment (asafMasa) [inline src/common/test-helpers/text-field.helper.spec.ts:71]: what is the purpose of this test?

File: src/exporter/components/export/export-dialog.spec.tsx [hunk 7]
```diff
@@ -175,8 +143,47 @@ describe('ExportDialog component', () => {
...
     );
 
-    updateField(wrapper, 'packageName', exportPackName);
-    updateField(wrapper, 'directoryName', exportDirName);
+    await waitFor(() => {
+      updateField(wrapper, 'packageName', exportPackName);
+      updateField(wrapper, 'directoryName', exportDirName);
+    })
+
+    // await updateFieldAsync(wrapper, 'packageName', exportPackName);
```
Comment (asafMasa) [inline src/exporter/components/export/export-dialog.spec.tsx:151]: ?

File: src/exporter/components/export/export-dialog.spec.tsx [hunk 8]
```diff
@@ -175,8 +143,47 @@ describe('ExportDialog component', () => {
...
+    // await updateFieldAsync(wrapper, 'directoryName', exportDirName);
+
+    wrapper.update();
+
+    await waitFor(() => {
+      const okButton = getButtonById(wrapper, 'general.ok-btn.text');
+      // const downloadLink = wrapper.find('#exportDownloadLink').text();
```
Comment (asafMasa) [inline src/exporter/components/export/export-dialog.spec.tsx:158]: ?

File: src/exporter/components/export/export-dialog.spec.tsx [hunk 9]
```diff
@@ -175,8 +143,47 @@ describe('ExportDialog component', () => {
...
+
+      expect(okButton.prop('disabled')).toBe(false);
+      // expect(downloadLink).toContain(exportPackName);
```
Comment (asafMasa) [inline src/exporter/components/export/export-dialog.spec.tsx:161]: ?

**Review by asafMasa — CHANGES_REQUESTED** [review]
🚀

File: src/common/test-helpers/button.helper.spec.ts (hunk 1 above)
Comment (alebinson) [reply src/common/test-helpers/button.helper.spec.ts:15]: It's for store helpers in *.spec files and not include them in build.

File: src/common/test-helpers/general.helper.spec.tsx (hunk 3 above)
Comment (alebinson) [reply src/common/test-helpers/general.helper.spec.tsx:48]: It's for store helpers in *.spec files and not include them in build. There are 3 helpers: general, textfield and button

File: src/common/test-helpers/text-field.helper.spec.ts (hunk 4 above)
Comment (alebinson) [reply src/common/test-helpers/text-field.helper.spec.ts:44]: Please leave it it's a full updated field cylcle, but currently not used.

File: src/common/test-helpers/text-field.helper.spec.ts (hunk 5 above)
Comment (alebinson) [reply src/common/test-helpers/text-field.helper.spec.ts:57]: Same as previous
1 focus
2 update
3 esc
4 blur

File: src/common/test-helpers/text-field.helper.spec.ts (hunk 6 above)
Comment (alebinson) [reply src/common/test-helpers/text-field.helper.spec.ts:71]: It's for store helpers in *.spec files and not include them in build. There are 3 helpers: general, textfield and button

File: src/common/test-helpers/button.helper.spec.ts (hunk 2 above)
Comment (alebinson) [reply src/common/test-helpers/button.helper.spec.ts:19]: Done, think it's already irrelevant to check last line

File: src/exporter/components/export/export-dialog.spec.tsx (hunk 7 above)
Comment (alebinson) [reply src/exporter/components/export/export-dialog.spec.tsx:151]: Removed

File: src/exporter/components/export/export-dialog.spec.tsx (hunk 8 above)
Comment (alebinson) [reply src/exporter/components/export/export-dialog.spec.tsx:158]: Removed, remains from prev version

File: src/exporter/components/export/export-dialog.spec.tsx (hunk 9 above)
Comment (alebinson) [reply src/exporter/components/export/export-dialog.spec.tsx:161]: Removed

Comment (alebinson) [reply src/exporter/components/export/export-dialog.spec.tsx:161]: Whole test removed, duplication

**Review by asafMasa — APPROVED** [review]

**Pull request merged**